.travis.yml

```diff
@@ -14,17 +23,157 @@ before_script:
   - mysql -uroot -e "GRANT ALL PRIVILEGES ON otptest.* TO otptest@localhost;"
   - mysql -uroot -e "CREATE USER otptest2@localhost IDENTIFIED BY 'otptest2';"
   - mysql -uroot -e "GRANT ALL PRIVILEGES ON otptest.* TO otptest2@localhost;"
-  - mysql -uroot -e "CREATE USER otptestssl@localhost IDENTIFIED BY 'otptestssl';"
-  - mysql -uroot -e "GRANT ALL PRIVILEGES ON otptest.* TO otptestssl@localhost REQUIRE SSL;"
+  - (mysql -uroot -e "CREATE USER otptestssl@localhost IDENTIFIED BY 'otptestssl' REQUIRE SSL;" &&
+    mysql -uroot -e "GRANT ALL PRIVILEGES ON otptest.* TO otptestssl@localhost;") ||
+    (mysql -uroot -e "CREATE USER otptestssl@localhost IDENTIFIED BY 'otptestssl';" &&
+    mysql -uroot -e "GRANT ALL PRIVILEGES ON otptest.* TO otptestssl@localhost REQUIRE SSL;")
```

Comment:
Wow. May I assume this the only way to make this work in all versions of MySQL? (A code comment explaining that wouldn't hurt.)

Reply:
Probably not the _only_ way but the easiest/quickest I came up with, as my main goal was to get all MySQL versions running, sort of as a proof of concept, so I left this part in a temporary mess ;)